recipe/meta.yaml

```diff
@@ -56,6 +56,7 @@ outputs:
         - libsentencepiece
         - libprotobuf
         - libabseil
+        - pcre2
```

Comment:
is it required? 
It's built as a part of Tokenizers build and not used as external dependency.